transform/mattermost-analytics/models/intermediate/data_eng/signup/_int_signup__models.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
```

Comment:
A better place to place signup related models can be under the name of the team responsible for this flow.